Cargo.toml

```
@@ -12,7 +12,7 @@ include = ["src/**/*", "Cargo.toml"]
 
 [dependencies]
 bytes = "0.4"
-protobuf = "2.2.0"
+protobuf = "2.6.1"
```

shouldn't this be the latest version instead? 
```suggestion
protobuf = "2.6.2"
```

https://github.com/stepancheg/rust-protobuf/issues/411#issuecomment-498098502